style.css

```diff
@@ -0,0 +1,167 @@
+.body {
+    max-width: 1920px;
+    margin: 0 auto;
+    background: url('pic/iswanto-arif-448003-unsplash.png') no-repeat;
+    background-size: cover;
+    overflow: hidden;
+    height: 100vh;
+    width: 100%;
+}
+.header {
+    width: 100%;
+    padding: 89px 114px 0 200px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    box-sizing: border-box;
+    margin: 0 0 97px 0;
+}
+.container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 0 0 200px;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Indonesia/pullRequest?prid=5674203)